blazor/pager/navigation.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: Navigation in Blazor Pager Component | Syncfusion
+description: Checkout and learn here all about available methods in Syncfusion Blazor Pager component and much more.
+platform: Blazor
+control: Pager
+documentation: ug
+---
+
+# Navigation in Pager Component
```

Comment:
Please get the updated methods API name from below file
https://github.com/essential-studio/ej2-blazor-source/blob/43bb65bf059572e9058cd24cbfb805c66202ebba/Syncfusion.Blazor/Navigations/Pager/SfPager.razor.method.cs